### ISSUE TITLE
Uploading CSV file creates duplicate table

### ISSUE THREAD

**Issue by MerielP** [ox-it/ords#765]
I've just created a new ORDS database and uploaded a CSV file to it. I was a little surprised to open the schema designer and find not one, but two tables:

![duplicate tables](https://cloud.githubusercontent.com/assets/13660306/13120392/8df28f24-d5a5-11e5-9254-ac858f3febb1.PNG)

This is on Dev in the project called 'Sample project' - Dave, you're already a project admin. The database is called 'Artefact catalogue'.

This happened after I'd had some difficulty getting the file to upload in the first place: I first tried it in another (old) database, and just got an error message. (I'll do a bit more testing and see if I can reproduce that error when I'm next in.)

**Comment by MerielP** [comment]
A bit of further investigation indicates that this problem seems confined to Dev - it didn't happen when I uploaded the same file to Staging and App.

**Comment by thestoat** [comment]
I have just tried this and the problem didn't happen for me. Please attach the file that causes the problem (or send it to me)

**Comment by MerielP** [comment]
While this was happening when I first created the issue, it doesn't seem to be happening now. I've just tried uploading the file that caused the problem to both a new database and an existing one, and both times it was fine.

Consequently, I'm going to work on the assumption that other changes made to the code in the last couple of months have sorted this out, and close the issue. (I'll reopen if it happens again.)
